docs/source/i18n/ultimate.es.rst

```diff
@@ -19,10 +19,30 @@ Si cambia de opinión y retira su consentimiento para participar en este estudio
 
 Al aceptar participar, estará haciendo una donación gratis y generosa para la investigación que podría ayudar a otros. Es posible que partes de la investigación llevada al cabo usando su información finalmente podría llevar al desarrollo de nuevos métodos para estudiar el cerebro, nuevas pruebas de diagnóstico, nuevos fármacos u otros productos comerciales. Si esto ocurre, no hay plan para proporcionarle a usted ninguna parte de los beneficios de estos productos y usted no tendrá ningún derecho de propiedad a estos productos.
 
-A nuestro entender, los datos que publicamos en abierto en general no contendrán información que puede identificarle directamente. La Base de Datos generada no contendrá su nombre, solo un código, de forma que el público no sabrá ni su nombre ni que los datos son suyos. Además, los datos no van a incluir información que pensemos podrían ayudar a personas cercanas a usted a adivinar qué los datos son suyos, como rasgos faciales o el día que participo. Si escribimos un reportaje o artículo sobre el estudio o compartimos los datos con otros, lo haremos de manera que usted no podrá ser identificado directamente. Sin embargo, usando datos adicionales relacionados con su nombre (por ejemplo, un escaneo cerebral obtenido de su historia clínica), uno posiblemente podría asociar su imagen u otra información en nuestra base de datos con usted. Además, si se produce una violación de seguridad (ataque cibernético) que pudiera hacer qué alguien le asociarse a usted con sus datos. Este riesgo es muy bajo porque sus datos están guardados en una base de datos segura, y la información sobre usted está guardado por separado, relacionados solo con un código.
+Desde nuestro conocimiento y mejor entender, los datos que publicamos en abierto en general no contendrán información que pueda identificarlo directamente. La Base de Datos generada no contendrá su nombre, solo un código, de forma que el público no sabrá ni su nombre ni que los datos son suyos. Además, los datos no van a incluir información que pensemos podrían ayudar a personas cercanas a usted a adivinar qué los datos son suyos, como rasgos faciales o el día que participo. Si escribimos un reportaje o artículo sobre el estudio o compartimos los datos con otros, lo haremos de manera que usted no podrá ser identificado directamente. Sin embargo, usando datos adicionales relacionados con su nombre (por ejemplo, un escaneo cerebral obtenido de su historia clínica), uno posiblemente podría asociar su imagen u otra información en nuestra base de datos con usted. Además, si se produce una violación de seguridad (ataque cibernético) que pudiera hacer qué alguien le asociarse a usted con sus datos. Este riesgo es muy bajo porque sus datos están guardados en una base de datos segura, y la información sobre usted está guardado por separado, relacionados solo con un código.
```

Comment:
```suggestion
Desde nuestro conocimiento y mejor entender, los datos que publicamos en abierto en general no contendrán información que pueda identificarlo directamente. La Base de Datos generada no contendrá su nombre, solo un código, de forma que el público no sabrá ni su nombre ni que los datos son suyos. Además, los datos no van a incluir información que pensemos podrían ayudar a personas cercanas a usted a adivinar qué datos son suyos, como rasgos faciales o el día que participo. Si escribimos un reportaje o artículo sobre el estudio o compartimos los datos con otros, lo haremos de manera que usted no podrá ser identificado directamente. Sin embargo, usando datos adicionales relacionados con su nombre (por ejemplo, un escaneo cerebral obtenido de su historia clínica), uno posiblemente podría asociar su imagen u otra información en nuestra base de datos con usted. Además, si se produce una violación de seguridad (ataque cibernético) que pudiera hacer qué alguien le asociarse a usted con sus datos. Este riesgo es muy bajo porque sus datos están guardados en una base de datos segura, y la información sobre usted está guardado por separado, relacionados solo con un código.
```

this is the same phrasing as is used in the second part

Reply:
```suggestion
Desde nuestro conocimiento y mejor entender, los datos que compartimos con el público general no contendrán información que pueda identificarlo directamente. Los datos que compartimos no contendrán su nombre, solo un código numérico, por lo que la gente no sabrá su nombre o qué datos son suyos. Además, los datos puestos a disposición del público en general no incluirán datos que pensamos que podrían ayudar a las personas que lo conocen a adivinar qué datos son suyos, como sus rasgos faciales o la fecha en que participó en el estudio. Si escribimos un informe o artículo sobre este estudio o compartimos el conjunto de datos del estudio con otras personas, lo haremos de tal manera que usted no pueda ser identificado directamente. Sin embargo, al utilizar datos adicionales vinculados a su nombre (por ejemplo, escáneres cerebrales obtenidos de sus registros médicos), uno podría potencialmente relacionar sus imágenes u otra información en nuestra base de datos con usted. Además, una brecha de seguridad (intrusión o ciberataque) podría hacer que alguien pudiera asociarlo a sus datos. Este riesgo es muy bajo porque sus datos se almacenan en una base de datos segura y la información sobre su identidad se almacena de manera separada a los datos en sí, únicamente vinculados a través de un código.
```

Thanks! I've make it match to the translation in the second part.